[Mongodb Atlas] Organization Log Datastream (#9754)
elastic/integrations: packages/mongodb_atlas/_dev/build/docs/README.md

1. Public Key
2. Private Key
3. Group ID
4. Organization ID

### Steps to obtain Public Key, Private Key, Group ID, and Organization ID

1. Generate programmatic API keys with `Organization Owner` permission by following the instructions in the Atlas [documentation](https://www.mongodb.com/docs/atlas/configure-api-access/#grant-programmatic-access-to-an-organization). Then, copy the public and private keys which function as a username and API key respectively.

> 2. From the Atlas UI with `project owner` permission, go to Project Settings > Access Manager > API Keys and then click on Invite To Project to add the API key created above, as described in this Atlas [document](https://www.mongodb.com/docs/atlas/configure-api-access/#invite-an-organization-api-key-to-a-project).

Reviewer: UI text and components are bold
```suggestion
2. From the Atlas UI with `project owner` permission, go to *Project Settings* > *Access Manager* > *API Keys* and then click *Invite To Project* to add the API key created above, as described in the Atlas [documentation](https://www.mongodb.com/docs/atlas/configure-api-access/#invite-an-organization-api-key-to-a-project).
```
